python/cuxfilter/charts/core/aggregate/core_number_chart.py

```diff
@@ -5,6 +5,8 @@
 
 class BaseNumberChart(BaseChart):
     stride = 1
+    # widget is a chart type that can be rendered in a sidebar or main layout
```

Comment:
"widget is a chart type" is a strange comment to have to write. :)  I wonder if some conceptual refactoring is in order soon, e.g so that "charts" and "widgets" are both "components" (but widgets would not be charts)

Reply:
agreed, that's a good idea